spring-data-jpa-batch-dml/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# ===============================
+# = DATA SOURCE
+# ===============================
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.url=jdbc:mysql://localhost:3306/baeldung_db?rewriteBatchedStatements=true
```

Comment:
I think it would be better to use H2 here so that the user doesn't need to stand up a database to see your examples work.